packages/bvaughn-architecture-demo/components/inspector/PropertiesRenderer.tsx

```diff
@@ -113,7 +118,7 @@ export default function PropertiesRenderer({
       ))}
 
       <Suspense fallback={<Loader />}>
-        {properties.map((property, index) => (
+        {properties.slice(propertiesIndex).map((property, index) => (
```

Comment:
I don't think I understand what changed here exactly, I actually can't even tell if it's related to the change up above?

Reply:
It's unrelated, but I had a side effect in this code before (was calling `array.splice` instead of `array.slice`). Overall this was a some pretty clowny code on my park.